shaal/DrupalPod
# Install prophecy-phpunit:^2

# The Problem/Issue/Bug
`phpspec/prophecy-phpunit:^2` was not installed

## How this PR Solves The Problem
Installs `phpspec/prophecy-phpunit:^2` in `drupalpod-setup.sh` and in `create-environment.sh`
fixes https://github.com/shaal/DrupalPod/issues/77

## Manual Testing Instructions
Open DrupalPod workspace with the update from this PR -
https://gitpod.io/#DP_PROJECT_NAME=single_content_sync,DP_ISSUE_FORK=single_content_sync-3229471,DP_ISSUE_BRANCH=3229471-cover-functionality-with-tests-1.2.x,DP_PROJECT_TYPE=project_module,DP_MODULE_VERSION=1.2.x,DP_CORE_VERSION=9.3.x,DP_PATCH_FILE=,DP_INSTALL_PROFILE=minimal/https://github.com/shaal/drupalpod/tree/fix-phpunit

Run `ddev phpunit --group=single_content_sync`, confirm there's no error message.


@todo:
When opening default DrupalPod, `composer.json` should include
```json
 "require-dev": {
        "drupal/core-dev": "^9.3",
        "phpspec/prophecy-phpunit": "^2"
    }
```

Without `drupal/core-dev`, `vendor/behat` will not exist, thus phpunit will fail.

## Related Issue Link(s)

## Release/Deployment notes



<a href="https://gitpod.io/#https://github.com/shaal/DrupalPod/pull/78"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
